### PR TITLE
Add 404 page implementation. Make some additions in gitignore.

### PR DESCRIPTION
**Pull request by angtheod**
@comzeradd @koyan
Hello Nikos! I did the PR for you, please see the comments below from Mike.

### CODE REVIEW

File: src/scss/pages/_404.scss [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+      @include x-large-and-up {
+        margin-top: -200px;
+      }
+    }
+  }
+
+  .page-header {
+    color: $black;
+    min-height: 1200px;
```
Comment (mikeselander) [inline src/scss/pages/_404.scss]: Why the `min-height`?

File: src/scss/pages/_404.scss [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    padding-top: 80px;
+
+    @include medium-and-up {
+      min-height: 1200px;
```
Comment (mikeselander) [inline src/scss/pages/_404.scss]: This value matches the prior one so is unnecessary.

File: src/scss/pages/_404.scss [hunk 3]
```diff
@@ -0,0 +1,102 @@
+.page-404-page {
+
```
Comment (mikeselander) [inline src/scss/pages/_404.scss]: The 404 page has different gradient background values so you could update those here.

See https://github.com/greenpeace/planet4-presentation-layer/blob/master/src/scss/pages/post/post.scss#L4 for an example of this.

File: src/scss/pages/_404.scss [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        background: none;
+        border: none;
+        font-size: 20px;
+        position: absolute;
+        right: 15px;
+        top: 38px;
+      }
+    }
+  }
+}
```
Comment (mikeselander) [inline src/scss/pages/_404.scss]: Missing EOF line.

File: 404.html [hunk 5]
```diff
@@ -0,0 +1,321 @@
...
+<!-- Site Footer End -->
+
+<!-- JS Libs that we include -->
+<script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.11.0/umd/popper.min.js" integrity="sha384-b/U6ypiBEHpOf/4+1nzFpr53nxSS+GLCkfwBdFNTxtclqqenISfwAzpKaMNFNmj4" crossorigin="anonymous"></script>
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/js/bootstrap.min.js" integrity="sha384-h0AbiXch4ZDo7tp9hKZ4TsHbi047NrKGLO3SEJAg45jXxnGIfYzk4Si90RDIqNm1" crossorigin="anonymous"></script>
+
+<!-- File for all the custom JS that we write -->
+<script type="text/javascript" src="custom.js"></script>
+<!-- <script type="text/javascript" src="take-action.js"></script> -->
```
Comment (mikeselander) [inline 404.html]: No need for this commented code.

File: 404.html [hunk 6]
```diff
@@ -0,0 +1,321 @@
...
+<html>
+<head>
+	<meta charset="UTF-8" />
+	<title>404 :: Greenpeace 2017</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+	<link rel="shortcut icon" type="image/ico" href="favicon.ico" />
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-alpha.6/css/bootstrap.min.css" integrity="sha384-rwoIResjU2yc3z8GV/NPeZWAv56rSmLldC3R/AZzGRnGxQQKnKkoFVhFQhNUwEyJ" crossorigin="anonymous">
+	<link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body class="brown-bg page-404-page">
```
Comment (mikeselander) [inline 404.html]: Looks from the designs like this should be a white background, and not brown.

File: 404.html [hunk 7]
```diff
@@ -0,0 +1,321 @@
...
+				We're sorry it looks like the page your looking for
+				isn't there. View our 404 check list and either try
+				again or enter the pages key word our search tool.
+			</p>
+		</div>
+		<h3 class="page-header-subtitle">Check list</h3>
+		<p class="page-header-content">
+		<ul>
+			<li>Point One Lorem ipsum dolor sit amet,</li>
+			<li>Point Two Lorem ipsum dolor sit amet,</li>
```
Comment (mikeselander) [inline 404.html]: The font and colors on this text doesn't match the design.

File: src/scss/pages/_404.scss [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+    color: $black;
+    min-height: 1200px;
+    padding-top: 80px;
+
+    @include medium-and-up {
+      min-height: 1200px;
+      padding-top: 110px;
+    }
+
+    .speech-bubble {
```
Comment (mikeselander) [inline src/scss/pages/_404.scss]: Could this just be on its own instead of nested?

File: src/scss/pages/_404.scss (hunk 4 above)
Comment (angtheod) [reply src/scss/pages/_404.scss]: @mikeselander 
Hi Mike, I am sorry for the confusion.I should have pointed out that this work was done by @comzeradd here https://github.com/greenpeace/planet4-test-presentation-layer-yv8v/commit/f1e3321cc87f50f30eb990224e280b5ac438e92d

I just did the PR for him. Hopefully he will see your comments soon and reply!

**Comment by comzeradd** [comment]
Thanks both for the PR and the review. Since I don't have permissions to push on this branch I created a new PR at #133. I rebased the code so we don't have merge commits and squashed everything to one commit.

**Comment by comzeradd** [comment]
We can close this one, since #133 was merged.

**Pull request closed without merging**